src/mame/jpm/jpmsru.cpp

```diff
@@ -509,6 +510,11 @@ void jpmsru_state::out_logicext_w(offs_t offset, uint8_t data)
 	}
 }
 
+void jpmsru_state::out_loudspeaker_w(offs_t offset, uint8_t data)
+{
+	m_dac->write(data);
+}
+
```

Comment:
Is this trampoline necessary?  Can you not just install the DAC’s write handler in the memory map with the appropriate `umask`?

Reply:
Fixed.